src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,18 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+
+export const CommentInfo = ({ comment }) => (
+  <div className="CommentInfo">
```

Comment:
it's a good practice to use object destructing 
```suggestion
export const CommentInfo = ({ comment }) => {
  const {...} = comment
  
  return (
    <div className="CommentInfo">
      ....
```